### DISCUSSION TITLE
Listen/Respond to commands from multiple Discord channels

### DISCUSSION THREAD

**Discussion by P1xellat3d** [cameroncros/OctoPrint-DiscordRemote#216] [Ideas]
Hi there,

I was wondering if it is at all possible to expand the capabilities of this plug-in to listen/send to multiple Discord channels (for example, two different channels from two different Discord servers)?

**Comment by cameroncros** [comment]
This is almost certainly possible, and someone was working on it prior to a rewrite. I dont have the time to make the changes myself, but if someone else makes the change I am more than happy to merge it. I am also happy to give any pointers if someone needs help to make the change.
